client-misk/src/test/kotlin/app/cash/backfila/client/misk/ClientMiskTestingModule.kt

```diff
@@ -0,0 +1,59 @@
+package app.cash.backfila.client.misk
+
+import app.cash.backfila.client.misk.client.BackfilaClientConfig
+import app.cash.backfila.client.misk.embedded.EmbeddedBackfilaModule
+import misk.MiskTestingServiceModule
+import misk.environment.Environment
+import misk.environment.EnvironmentModule
+import misk.hibernate.HibernateEntityModule
+import misk.hibernate.HibernateModule
+import misk.hibernate.HibernateTestingModule
+import misk.inject.KAbstractModule
+import misk.jdbc.DataSourceConfig
+import misk.jdbc.DataSourceType
+import misk.logging.LogCollectorModule
+import kotlin.reflect.KClass
+
+internal class ClientMiskTestingModule(
+  val useVitess: Boolean,
+  val backfillClasses: List<KClass<out Backfill<*, *>>>
+) : KAbstractModule() {
+  override fun configure() {
+    val dataSourceConfig =  when {
+      useVitess ->  DataSourceConfig(
```

Comment:
formatting

Reply:
Not 100% sure I know what you are looking for. But I autoformatted.